olp-cpp-sdk-core/include/olp/core/utils/LruCache.h

```diff
@@ -146,10 +237,10 @@ class LruCache {
         max_size_(maxSize),
         size_(0) {}
 
-  /// deleted copy constructor
+  /// A deleted copy constructor.
```

Comment:
It is rather `the deleted copy constructor` as `A` implies there are more of these but actually you only have one.

Reply:
Changed.